### PR TITLE
Add integration test with kvm enabled

### PR DESCRIPTION
**Pull request by cqs21**

### CODE REVIEW

File: .github/workflows/test_asterinas.yml [hunk 1]
```diff
@@ -86,3 +86,53 @@ jobs:
       - name: General Test (Linux EFI Handover Boot Protocol)
         id: test_linux
         run: make run AUTO_TEST=test ENABLE_KVM=0 BOOT_PROTOCOL=linux-efi-handover64 RELEASE=1
+
+  integration-test-with-kvm-enabled:
```
Comment (tatetian) [inline .github/workflows/test_asterinas.yml:90]: We need to balance between the coverage and cost of testing. Ideally, we want to test every combination of configurations such as KVM enabled/disabled, release/debug builds, and boot protocols. But there are too many combinations. Running the entire integration test with KVM enabled again after doing it already with KVM disabled is not cost effective. I suggest running all tests with KVM enabled by default and only disabling KVM for a selected set of tests. One strategy is to disable KVM only when running "Syscall Test at Exfat".

File: .github/workflows/test_asterinas.yml [hunk 2]
```diff
@@ -42,47 +42,49 @@ jobs:
...
         id: syscall_test
-        run: make run AUTO_TEST=syscall ENABLE_KVM=0 BOOT_PROTOCOL=linux-efi-handover64 RELEASE=1
+        run: make run AUTO_TEST=syscall ENABLE_KVM=1 BOOT_PROTOCOL=linux-efi-handover64 RELEASE=1
 
       - name: Syscall Test at Ext2 (MicroVM)
         id: syscall_test_at_ext2
-        run: make run AUTO_TEST=syscall SYSCALL_TEST_DIR=/ext2 ENABLE_KVM=0 SCHEME=microvm RELEASE=1
+        run: make run AUTO_TEST=syscall SYSCALL_TEST_DIR=/ext2 ENABLE_KVM=1 SCHEME=microvm RELEASE=1
 
       - name: Syscall Test at Exfat
```
Comment (tatetian) [inline .github/workflows/test_asterinas.yml:81]: Need to emphasize that this test has disabled KVM.

```
      - name: Syscall Test at Exfat and without KVM enabled
```

**Comment by cqs21** [comment]
Should we also disable KVM when running "Syscall Test at Ext2"?

**Comment by cqs21** [comment]
>  Could you investigate why "Syscall Test at Ext2" failed when KVM is enabled?

I cannot reproduce it on the local machine, maybe it's also caused by #970.

**Review by tatetian — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**